### ISSUE TITLE
Mobile Site

### ISSUE THREAD

**Issue by JapanYoshi** [kayteh/roleypoly#5]
The current site has a mobile-unfriendly layout, and forbids zooming out to see the full picture.

Assigning roles to groups is impossible on mobile devices, as it requires dragging and dropping.

As many Discord users only use it on its smartphone app, mobile support is an indispensable feature that's currently missing.

**Comment by kayteh** [comment]
I agree, I'm in the middle of a slow rewrite/refresh since my $dayjob is eating all of my time for now, this is absolute top priority.

**Comment by JapanYoshi** [comment]
Alright, good to know! This might want to be a separate issue, but I also think it would be neat to have server-specific background colors (and choice between dark and light text colors).

**Comment by kayteh** [comment]
That sounds interesting enough. Will consider!

**Comment by JapanYoshi** [comment]
Bump.

**Comment by kayteh** [comment]
It's in progress on `develop` atm, taking me a good bit of time, but I'm getting there.

**Comment by JapanYoshi** [comment]
Hope this is coming along nicely.
